index.html

```diff
@@ -74,6 +75,220 @@ <h1 class="main-text">I’m Giovanni Glad to see you!</h1>
+                
+                <div>
+                    <p class="about-secondary-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>      
+            </div>
+
+            <div class="about-social-container">
+                <div class="about-connect-title">LET'S CONNECT</div>
+                    <div>
+                        <ul class="connect-icons">
+                            <li>
+                                <a href="">
+                                    <img class="twitter" src="./Assets/twitter.svg" alt="Connect Twitter">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="Linkedin" src="./Assets/Linkedin icon.svg" alt="Connect Linkedin">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="medium" src="./Assets/medium.svg" alt="Connect Medium">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="github" src="./Assets/github.svg" alt="Connect Github">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="angellist" src="./Assets/angellist.svg" alt="Connect Angel List">
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="button-resume-div">
+                    <button class="button-resume">Get my Resume</button>
+                </div>
+
+                <div class="languages">
+                    <div class="language-title">
+                        <p>Languages</p>
+                    </div>
+                    <div class="languages-list">
+                        <ul>
+                            <li class="languages-li">
+                                <img src="/Assets/javascript_circle.png" alt="">
+                                <p>JAVASCRIPT</p>
+                            </li>
+                            <li class="languages-li">
+                                <img src="/Assets/html_circle.png" alt="">
+                                <p>HTML</p>
+                            </li>
+                            <li class="languages-li">
+                                <img src="/Assets/css_circle.png" alt="">
+                                <p>CSS</p>
```

Comment:
- [ ] Hey @gijoe2day 👋 You have done excellently well in implementing some of the changes suggested by the previous reviewer. But It would be nice if you fix the images in this section by adding the correct image path so that it aligns with the Figma template.

> Figma Template

![Figma Template](https://user-images.githubusercontent.com/71140133/198749781-5fd8b732-cca5-4562-99ce-55dfdfcac73a.png)

> Your design

![Your design](https://user-images.githubusercontent.com/71140133/198749917-7cbecee4-6e06-4d3d-8764-fae4d3460258.png)

Reply:
hi @Yerimah.  But thats really weird cause in me desktop it appears.  Please check this loom video: https://www.loom.com/share/c9a80ad73c3049a680710e71e54f94dc

Reply:
hi @HAMMAS-SALEEM, @Yerimah.  Please help me reviewing the loom video as the images do appear in my end.  Hope it can be approve as all suggestions were fixed and are working as it is shown in the video.  Thx! https://www.loom.com/share/c9a80ad73c3049a680710e71e54f94dc

Reply:
Hi @Yerimah @HAMMAS-SALEEM, please see loom video on this issue, as images do appear perfectly in my end, well positioned and everything.

Reply:
Dear @gijoe2day, This might happen for a lot of reasons. when you face such a problem try to deploy your project using the GitHub pages. And submit a live link. This way you can be easily on the same page with your code reviewers.